Formula/gnome-builder.rb

```diff
@@ -34,7 +34,7 @@ class GnomeBuilder < Formula
   depends_on "ctags" => :recommended
   depends_on "meson" => :recommended
   depends_on :python3 => :optional
-  depends_on "pygobject3" if build.with? "python3"
+  depends_on "py3gobject3" if build.with? "python3"
```

Comment:
This could perhaps be made just `depends_on "py3gobject3" => :optional` (and similarly below)